test_scripts/Polices/Validation_of_PolicyTables/229_preloaded_pt_exists.lua

```diff
@@ -58,13 +61,13 @@ end
 
 --[[ Test ]]
 commonFunctions:newTestCasesGroup("Test")
+function Test.TestStep_start_sdl()
+  StartSDL(config.pathToSDL, config.ExitOnCrash)
+  os.execute("sleep 5")
+end
+
 function Test:TestStep_CheckSDLStops()
-  --TODO(istoimenova): Should be checked when ATF problem is fixed with SDL crash
-  --EXPECT_HMINOTIFICATION("BasicCommunication.OnSDLClose")
-  local result = testCasesForPolicySDLErrorsStops:CheckSDLShutdown(self)
-  if (result == false) then
-    self:FailTestCase("Error: SDL doesn't stop.")
-  end
+  testCasesForExternalUCS.checkSDLStatus(self, sdl.STOPPED)
 end
 
 function Test:TestStep_CheckSDLLogError()
```

Comment:
@istoimenova, Please add check on "BasicCommunication.OnSDLClose" in order to conform to other scripts